### PR TITLE
Data Explorer indicator & chart type controls

### PR DESCRIPTION
**Pull request by danielgavrilov**
Based off of `explorer-table` branch. That branch should be merged and this one rebased before merging.

Improvements:
- View layout / window resizing (https://www.notion.so/owid/View-layout-window-resizing-cd2c0a1371ed4e1aab0c1a847ddb0fa3)
- Updated design of indicator & chart type controls

To-do:
- [ ] Fix tests

### CODE REVIEW

File: charts/IndicatorDropdown.tsx [hunk 1]
```diff
@@ -26,47 +76,256 @@ export class IndicatorDropdown extends React.Component<IndicatorDropdownProps> {
...
+            <h3 className="title">{indicator.title}</h3>
+            {indicator.subtitle && (
+                <p className="subtitle">{indicator.subtitle}</p>
+            )}
+            {indicator.sourceDesc && (
+                <p className="metadata">{indicator.sourceDesc}</p>
+            )}
+        </div>
+    )
+}
```
Comment (MarcelGerber) [inline charts/IndicatorDropdown.tsx]: Maybe you can "just" make use of `formatOptionLabel`, like I did [for the color scheme dropdown](https://github.com/owid/owid-grapher/blob/18d1a6d8eec9d4d28ce17efba66935120d4d593e/admin/client/ColorSchemeDropdown.tsx#L83-L113)?

The advantage is that you don't have to care about stuff like classes for focused, selected and stuff. The disadvantage is that you're probably more restricted.

File: charts/IndicatorDropdown.tsx [hunk 2]
```diff
@@ -26,47 +76,256 @@ export class IndicatorDropdown extends React.Component<IndicatorDropdownProps> {
...
-    @bind getLabel(indicator: Indicator): string {
-        if (indicator.sourceDesc) {
-            return `${indicator.title}, ${indicator.sourceDesc}`
+    @computed get filteredIndicators(): Indicator[] {
+        if (this.search) {
+            return this.indicators.filter(
+                indicator =>
+                    (indicator.title || "")
+                        .toLocaleLowerCase()
+                        .indexOf(this.search.toLowerCase()) > -1
```
Comment (MarcelGerber) [inline charts/IndicatorDropdown.tsx]: You could use the more modern [`String.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) here

**Comment by MarcelGerber** [comment]
Also, I get two scrollbars for the dropdown
image

**Comment by jasoncrawford** [comment]
Closing for now as per @danielgavrilov

**Pull request closed without merging**